### PR TITLE
BE - Asset listing - Implement api

### PR DESCRIPTION
**Pull request by ruslanpalagin**
https://app.clickup.com/t/3epp6pc

### CODE REVIEW

**Review by dmitriy-green — APPROVED** [review]

File: api/src/features/assets/asset.entity.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  @Column()
+  propertyManagementPercentage: number;
+
+  @Column()
+  utilitiesPercentage: number;
+
+  @Column()
+  llcAdministrationFeePercentage: number;
+
+  @Column()
```
Comment (boivlad) [inline api/src/features/assets/asset.entity.ts:95]: Are these two columns created automatically if they are not disabled, or was there some special idea to write them explicitly?

File: api/src/features/assets/assets.controller.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  async create(
+    @Req() request: Request,
+    @Res({ passthrough: true }) response: Response
+  ): Promise<any> {
+      const attrs = request.body.attrs
+      const { isValid, errors } = await validate(attrs)
+      if (!isValid) {
+        response.status(HttpStatus.UNPROCESSABLE_ENTITY).json({ errors })
+      } else {
+        const resource = await this.assetsRepository.save(attrs)
```
Comment (boivlad) [inline api/src/features/assets/assets.controller.ts:33]: As for me, it is better to move the logic of working with the database, etc., except for the validation of the request attributes, to the service file and add additional checks, including `try/catch` to catch errors when working with the database

**Review by boivlad — APPROVED** [review]

File: api/src/features/assets/assets.controller.ts (hunk 2 above)
Comment (ruslanpalagin) [reply api/src/features/assets/assets.controller.ts:33]: 1. Basically yes, but here it was an empty layer so I used repo in controller. I know this is unusual a bit.
2. I checked errors handling - it works

File: api/src/features/assets/asset.entity.ts (hunk 1 above)
Comment (ruslanpalagin) [reply api/src/features/assets/asset.entity.ts:95]: No, not automatic. 
According to the doc I see it's a regular data storage field. Let's discuss

**Pull request merged**